torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -122,13 +123,30 @@ def resize_image_tensor(
     if image.numel() > 0:
         image = image.reshape(-1, num_channels, old_height, old_width)
 
-        image = _FT.resize(
+        align_corners: Optional[bool] = None
+        if interpolation == InterpolationMode.BILINEAR or interpolation == InterpolationMode.BICUBIC:
+            align_corners = False
+        elif antialias:
+            raise ValueError("Antialias option is supported for bilinear and bicubic interpolation modes only")
```

Comment:
Shouldn't we have this outside of the `if` branch to always raise this error even if we have degenerate shapes?

Reply:
yeah good call, I'll move it out.